deps/media-playback/media-playback/media.c

```diff
@@ -811,8 +812,6 @@ void mp_media_free(mp_media_t *media)
 	os_sem_destroy(media->sem);
 	sws_freeContext(media->swscale);
 	av_freep(&media->scale_pic[0]);
-	bfree(media->path);
-	bfree(media->format_name);
```

Comment:
Moving these `bfree` calls to the obs-ffmpeg source code doesn't seem right.  Why do you need to do this?

Reply:
I confirmed in my own testing that this change introduces memory leaks.  Reverting this change, and the associated changes in `obs-ffmpeg` removed the memory leaks and still allowed media sources to reconnect.

Reply:
Because when mp_media_thread thread close, it will call mp_media_free to free media context, So that reconnect thread dons't get media context, So i want when the thread close dons't free context Unless the media source update.
can you give any suggest?

Reply:
> Moving these `bfree` calls to the obs-ffmpeg source code doesn't seem right. Why do you need to do this?

Because when mp_media_thread thread close, it will call mp_media_free to free media context, So that reconnect thread dons't get media context, So i want when the thread close dons't free context Unless the media source update.
can you give any suggest?

Reply:
@Yohox I don't really understand what you're seeing here and I can't see why this is needed at all. What goes wrong if you keep this where it is?